agent/src/agent/source/validator.py

```diff
@@ -250,17 +251,17 @@ class ObserviumValidator(Validator):
     VALIDATION_SCHEMA_FILE = 'observium.json'
 
     def validate_connection(self):
-        session = http.Session()
-        session.auth = (
-            self.source.config[source.ObserviumSource.USERNAME], self.source.config[source.ObserviumSource.PASSWORD]
-        )
         try:
-            url = urllib.parse.urljoin(self.source.config['url'], source.ObserviumSource.DEVICES_API_PATH)
-            res = session.get(url, verify=self.source.config.get('verify_ssl', True), timeout=self.source.query_timeout)
-            res.raise_for_status()
-        except requests.exceptions.RequestException as e:
+            MySQL(
+                self.source.config['host'],
+                self.source.config.get('port', 3306),
+                self.source.config.get('username'),
+                self.source.config.get('password'),
+                self.source.config['database'],
+            ).execute('show databases')
```

Comment:
maybe `show tables` will be better? the user may not have permissions to show all databases

Reply:
yep